paddle/trainer/TrainerConfigHelper.cpp

```diff
@@ -36,6 +36,22 @@ namespace paddle {
 
 struct TrainerConfigHelperPrivate {
   TrainerConfig conf;
+
+  /**
+   * @brief Update trainer default values to each Parameter.
+   */
+  void updateDefaultValues() {
+    // set default gradient clipping.
+    if (conf.default_values().has_gradient_clipping_threshold()) {
+      auto defaultGradientClipping =
+          conf.default_values().gradient_clipping_threshold();
+      for (auto &param : *conf.mutable_model_config()->mutable_parameters()) {
+        if (!param.has_gradient_clipping_threshold()) {
+          param.set_gradient_clipping_threshold(defaultGradientClipping);
+        }
+      }
+    }
+  }
```

Comment:
I cannot agree with this modification.

If we are not going to remove `config_parser.py`, the modification in this PR means a user cannot check and confirm the parameters he set in defining the task by print the protostr. How can I confirm whether `gradient_clipping` is enable if PaddlePaddle updates? No test guarantee this.